### ISSUE TITLE
UUID-Support

### ISSUE THREAD

**Issue by Phoenix616** [cppchriscpp/TravelPortals#16]
The plugin still runs on playernames which can be changed now. We need to convert the plugin and the names to UUIDs.

We might want to look into using plugins that already exist and store UUIDs on the server like Essentials, PermissionsEx or LogBlock before asking Mojang directly for the UUID of players or provide an option to only convert when a player logs in for the first time after the udpate.

**Comment by cppchriscpp** [comment]
Sigh, yeah, this has been needed for a very long time, but I haven't put any time into doing it. The feature works the way it is, and I haven't seen enough interest in this plugin to motivate me to jump on it. 

I like the idea of converting when a player logs in for the first time, or alternatively converting everyone at startup (with clear logging in case it takes a while!) and setting a flag to prevent it from happening again. 

I'll be clear and honest here; I don't think I will do this any day soon. I barely play minecraft at this point, and do not spend much time on my plugins. If someone feels inclined to take this to the next level and submits a PR, I'll be happy to make a new release and help test and whatever else is needed to help it along. 

Alternatively, this project was released under the MIT license -- you are (or anyone is) free to modify it to your heart's content and even release those modified versions. If you'd rather go that route, I won't be offended.

**Comment by Phoenix616** [comment]
Yeah, it isn't really that popular nowadays and no one seems to really need that feature. I might implement it myself if I find the time and/or muse for it.

To be honest I partly opened that issue to remind my self to implement that, but with the hope that maybe someone will come around and do it before me. It shouldn't be much of a hassle for me 'though as I already updated a couple plugins I needed to support UUIDs, hadn't used the per user portal rights on this one before.

And I probably wont be releasing a modified version as long as my modifications are just minor fixes and tweaks and not changing major mechanics of the plugin (which I don't intend for now) but we'll see what the future will bring.
